File: SDKs/beezup.com/2.0/typescript/src/sdk/models/shared/userlovindex.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { UserLovIndexLinks } from "./userlovindexlinks";



export class UserLovIndex extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=links" })
  links?: UserLovIndexLinks;
}

File: SDKs/beezup.com/2.0/python/src/sdk/models/shared/userlovindex.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import userlovindexlinks as shared_userlovindexlinks


@dataclass_json
@dataclasses.dataclass
class UserLovIndex:
    links: Optional[shared_userlovindexlinks.UserLovIndexLinks] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('links') }})
    
